shared/fs/common/path-item-action-container.js

```diff
@@ -37,8 +37,8 @@ const mapDispatchToProps = (dispatch: Dispatch, {path}: OwnProps) => ({
   copyPath: () => dispatch(ConfigGen.createCopyToClipboard({text: Types.pathToString(path)})),
   ...(isMobile
     ? {
-        _saveMedia: () => dispatch(FsGen.createSaveMedia({path})),
-        _shareNative: () => dispatch(FsGen.createShareNative({path})),
+        _saveMedia: () => dispatch(Constants.createSaveMedia({path})),
```

Comment:
I'd rather these be `FsGen.createSaveMedia({path, key: Constants.makeDownloadKey(path)})`, to make explicitly clear what the action is named.

Reply:
hmm why is using Constants not clear? They are named the same way. I was hoping to make it easier for container since all we really need is `path`.

Reply:
Hmm actually I'll try to make a helper function so we can still use FsGen here.